### ISSUE TITLE
Play quest games with gargoyle

### ISSUE THREAD

**Issue by nickbe** [garglk/garglk#286]
Is it possible to play quest games with gargoyle?

**Comment by cspiegel** [comment]
Gargoyle includes Geas, which is an interpreter for Quest games.  From what I can tell, the Quest file format has changed since Geas was written, so newer games won't play on Geas.  Some older competition games (e.g. Beam from 1996) do work.

The source code for the modern Quest interpreter looks to be available on Github (https://github.com/textadventures/quest), but it appears to be written in Visual Basic and/or C#, so there's no way it could be adapted for use in Gargoyle.  Because the source is available it'd be possible to reimplement it in a portable language, but that would take a lot of effort.  I don't currently have access to a Windows machine so I can't take a look at the interpreter, but it also may be that modern Quest isn't a good fit with Glk, depending on how many bells and whistles it has.

In short, it's possible that a modern portable Quest interpreter could be written, but it's unlikely it'd happen any time soon.

**Comment by nickbe** [comment]
Ok. Thanks
